packages/grid/x-grid/src/tests/rows.XGrid.test.tsx

```diff
@@ -265,6 +264,13 @@ describe('<XGrid /> - Rows', () => {
         this.skip();
       }
     });
+    beforeEach(() => {
+      clock = useFakeTimers();
+    });
+
+    afterEach(() => {
+      clock.restore();
+    });
```

Comment:
Could we scope the time mocking to only where we need them? Its impact is no negligible.